src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -107,6 +128,12 @@ public void postToldpcvOfLdpnrWithoutMementoDatetimeMustIgnoreBody(final String
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
         //same as previous but with LDP-NR
+        // create the versioned resource
+        final String newResource = createVersionedNonRDFResource(uri, info).body().asString();
+        // create a memento for the current time
+        final Response versionResponse = doPost(newResource + "/fcr:versions", "Send some random data here");
+        confirmPresenceOfLinkValue(MEMENTO_LINK_HEADER, versionResponse);
+        confirmPresenceOfVersionLocationHeader(versionResponse);
```

Comment:
Per the requirement being tested, I would expect this test to also be verifying that the state of the created memento does not include the `POST`ed body.